skeleton-es2016-asp.net5/src/skeleton-navigation-es2016-vs/gulpfile.js

```diff
@@ -1,3 +1,4 @@
+/// <binding BeforeBuild='clean, build-system, build-html, build-css, build-jspm' Clean='clean' ProjectOpened='watch' />
```

Comment:
Correct, by adding these all you have to do when you start the project is hit either the `Start Debugging` or `Start without Debugging` buttons and all these tasks will run. I wanted to just have the `build` task to run but there was a bug with the task that I could not figure out.